packages/aws-cdk-lib/aws-ecs/README.md

```diff
@@ -692,18 +692,26 @@ new ecs.ExternalService(this, 'Service', {
 
 ### Deployment circuit breaker and rollback
 
-Amazon ECS [deployment circuit breaker](https://aws.amazon.com/tw/blogs/containers/announcing-amazon-ecs-deployment-circuit-breaker/)
-automatically rolls back unhealthy service deployments without the need for manual intervention. Use `circuitBreaker` to enable
-deployment circuit breaker and optionally enable `rollback` for automatic rollback. See [Using the deployment circuit breaker](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/deployment-type-ecs.html)
-for more details.
+The Amazon ECS [deployment circuit breaker](https://aws.amazon.com/tw/blogs/containers/announcing-amazon-ecs-deployment-circuit-breaker/)
+automatically rolls back unhealthy service deployments, eliminating the need for manual intervention.
+
+Use `circuitBreaker` to enable the deployment circuit breaker which determines whether a service deployment
+will fail if the service can't reach a steady state.
+You can optionally enable `rollback` for automatic rollback.
+You can disable the circuit breaker by omitting the `circuitBreaker` property or setting `enabled` to `false`.
```

Comment:
Is this accurate that omitting the `circuitBreaker` property will disable the circuit breaker? I believe that was the original issue and I don't think this PR that has directly altered that behaviour.

Reply:
I think you're right. It appeared to be an unnecessary detail, so I removed it.